kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -39,49 +39,42 @@
         :layout8="{ span: 6 }"
         :layout12="{ span: 10 }"
       >
-        <TabsWithOverflow
+        <KListWithOverflow
```

Comment:
When I tested it I noticed we're missing the styles and click-ability of the section buttons. I believe this is because the TabsWithOverflow component wrapped everything within a `KTabsList`.

I think that you'll need to still migrate some of that logic to this component - such as the "activeTabId" and "@click" handler and possibly the "tabsWrapper" ref.

I think that it may work by wrapping the whole `<template #item="{ item }">` block with the `KTabsList` component without worrying about the `#tab` slot there -- but I'm not super sure on this.